File: ARRAYS/Shuffle the Array/Shuffle the Array.py
#PROBLEM LINK:- https://leetcode.com/problems/shuffle-the-array/

class Solution:
    def shuffle(self, nums, n): 
        v = []
        for i in range(n):
            v.append(nums[i])
            v.append(nums[i+n])
        return v

File: ARRAYS/Shuffle the Array/Shuffle the Array.cpp
/*
PROBLEM LINK:- https://leetcode.com/problems/shuffle-the-array/
*/

class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        int m = nums.size();
        vector<int> v(m);
        for(int i = 0, j = 0; i < n; i++, j+= 2)
        {
            v[j] = nums[i];
            v[j+1] = nums[i+n];
        }
        return v;
    }
};
